java/code/src/com/redhat/rhn/domain/server/Server.java

```diff
@@ -128,6 +129,7 @@
     private String hostname;
     private boolean payg;
     private MaintenanceSchedule maintenanceSchedule;
+    private boolean inMaintenanceMode;
```

Comment:
I think this is incomplete. Just a variable without setter and getter does not make sense.
For now I would remove the changes for this class